welcome/index.html

```diff
@@ -0,0 +1,352 @@
+            <h4>Live Webcam Stream</h4>
+            <p>Remotely monitor your cat with Node.JS, ffmpeg and a connected webcam.</p>
+          </a>
+        </li>
+        <li class="more">
+          <a href="https://github.com/experimental-platform/example-voice-recognition" target="_blank">
+            <span class="img-example img-example-voice-recognition"></span>
+            <span class="img-technology img-technology-nodejs"></span>
+            <h4>Voice Recognition</h4>
+            <p>A simple app that listens for voice commands. Build your own Amazon Echo.</p>
+          </a>
+        </li>
+      </ul>
+      <p class="examples__note">
+        And the best part: experimental platform makes it dead easy to combine them.
+      </p>
+    </div><!-- /.more-tutorials -->
+
+    <div class="credits section-content">
+      <h2 class="headline-content credits">
+        <span>
+          <span class="credits__made-by">Made with &hearts; by</span>
+          <a href="https://protonet.info" class="credits__protonet"></a>
+        </span>
+      </h2>
+      <ul class="creators">
+        <li class="creators__profile">
+          <a href="https://github.com/https://github.com/hinnerk" class="profile" target="_blank">
+            <div class="profile__portrait hexagon"><img src="img/hinnerk.jpg" alt="Hinnerk Hardt"></div>
+            <div class="profile__name">Hinnerk Hardt</div>
+          </a>
+        </li>
+        <li class="creators__profile">
+          <a href="https://github.com/tiff" class="profile">
+            <div class="profile__portrait hexagon"><img src="img/tiff.jpg" alt="Christopher Blum"></div>
+            <div class="profile__name">Christopher Blum</div>
+          </a>
+        </li>
+        <li class="creators__profile">
+          <a href="https://github.com/kdomanski" class="profile">
+            <div class="profile__portrait hexagon"><img src="img/kamil.jpg" alt="Kamil Domański"></div>
+            <div class="profile__name">Kamil Domański</div>
+          </a>
+        </li>
+        <li class="creators__profile">
+          <a href="https://github.com/markbeee" class="profile">
+            <div class="profile__portrait hexagon"><img src="img/markus.png" alt="Markus Ulsass"></div>
+            <div class="profile__name">Markus Ulsass</div>
+          </a>
+        </li>
+        <li class="creators__profile">
+          <a href="https://github.com/oem" class="profile">
+            <div class="profile__portrait hexagon"><img src="img/oem.jpg" alt="Ömür Özkir"></div>
+            <div class="profile__name">Ömür Özkir</div>
+          </a>
+        </li>
+        <li class="creators__profile">
+          <a href="https://github.com/paukul" class="profile">
+            <div class="profile__portrait hexagon"><img src="img/paukul.jpg" alt="Pascal Friederich"></div>
+            <div class="profile__name">Pascal Friederich</div>
+          </a>
+        </li>
+      </ul><!-- /.creators -->
```

Comment:
design feedback: consider putting the slight css grayscale (which is also used on other img on the page) on the photos as well